com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/IOSBundler.java

```diff
@@ -29,8 +32,11 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-import org.apache.commons.configuration.ConfigurationException;
-import org.apache.commons.configuration.plist.XMLPropertyListConfiguration;
+import org.apache.commons.configuration2.io.FileLocator;
+import org.apache.commons.configuration2.io.FileLocator.FileLocatorBuilder;
+import org.apache.commons.configuration2.io.FileLocatorUtils;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.commons.configuration2.plist.XMLPropertyListConfiguration;
```

Comment:
I think it'd be good to have some explanation of what the bug is, and how it's related to updating the java packages?

Reply:
Hm, I thought it's clear from the issue I created. Ok, I'll add a bit more info in PR too

Reply:
I added a short explanation in the PR description. Thank you.